extensions/github-authentication/src/env/browser/authServer.ts

```diff
@@ -1,12 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-export function startServer(_: any): any {
```

Comment:
we should probably leave the extension we've forked from vscode as-is.

Reply:
Restored those files.